### ISSUE TITLE
Import database from 3.0.30 to 3.1.0 fails

### ISSUE THREAD

**Issue by JeffJochems** [orientechnologies/orientdb#9287]
## Expected behavior  
I want to import my database exported from a previous version of OrientDB (3.0.30) into OrientDB 3.1 . 

## Actual behavior  

When importing a database exported from orientDB 3.0.30, an error occurs with the following exception: ` Impossible to remove the last cluster of class 'ORole' drop the class instead`.

The full console response is:

`orientdb {db=new_database}> import database C:\old_database.gz`

`Importing database database C:\old_database.gz...`
`Error: com.orientechnologies.orient.core.exception.OStorageException: Error sending import request`
        `DB name="new_database"`

`Error: com.orientechnologies.orient.core.db.tool.ODatabaseExportException: Error on importing database 'new_database' from file: /tmp/import4863700426407477998data.gz`

`Error: com.orientechnologies.orient.core.exception.ODatabaseException:  Impossible to remove the last cluster of class 'ORole' drop the class instead`
        `DB name="new_database"`

This issue was also identified and raised on discourse [here](https://discourse.orientdb.org/t/import-database-fails-on-3-1-0-with-export-from-3-0-14/1195). Migrating databases from OrientDB 3.0.31 and 3.0.14 thus also seems to be causing this problem.

**Comment by dullahan8** [comment]
I can replicate this error trying to export from 3.0.25 and import into 3.1.0.
Orientdb: 3.1.0
OS: CentOS Linux release 7.6.1810 (Core)
Java: openjdk version "1.8.0_191"
```- Creating cluster 'orole'...Error on database import happened just before line 0, column 478
com.orientechnologies.orient.core.exception.ODatabaseException:  Impossible to remove the last cluster of class 'ORole' drop the class instead
        DB name="production-sync-dev"
        at com.orientechnologies.orient.core.metadata.schema.OClassEmbedded.removeClusterId(OClassEmbedded.java:557)
        at com.orientechnologies.orient.core.db.document.ODatabaseDocumentAbstract.dropCluster(ODatabaseDocumentAbstract.java:856)
        at com.orientechnologies.orient.core.db.tool.ODatabaseImport.importClusters(ODatabaseImport.java:982)
        at com.orientechnologies.orient.core.db.tool.ODatabaseImport.importDatabase(ODatabaseImport.java:177)
        at com.orientechnologies.orient.console.OConsoleDatabaseApp.importDatabase(OConsoleDatabaseApp.java:2055)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at com.orientechnologies.common.console.OConsoleApplication.execute(OConsoleApplication.java:451)
        at com.orientechnologies.common.console.OConsoleApplication.executeCommands(OConsoleApplication.java:287)
        at com.orientechnologies.common.console.OConsoleApplication.run(OConsoleApplication.java:142)
        at com.orientechnologies.orient.console.OConsoleDatabaseApp.main(OConsoleDatabaseApp.java:130)

Error: com.orientechnologies.orient.core.db.tool.ODatabaseExportException: Error on importing database 'production-sync-dev' from file: /tmp/production-2020-06-11.gz

Error: com.orientechnologies.orient.core.exception.ODatabaseException:  Impossible to remove the last cluster of class 'ORole' drop the class instead
        DB name="production-sync-dev"
```

**Comment by luigidellaquila** [comment]
Hi all

The problem is in the 3.1 importer and it applies to all previous versions. We have a fix and we are testing it, it will likely be available in the snapshot in a few hours

Thanks

Luigi

**Comment by everette00** [comment]
Awesome! Thanks @luigidellaquila !

**Comment by luigidellaquila** [comment]
Hi all

We pushed a fix to develop branch, the 3.1.1-SNAPSHOT version is already available on Sonatype 

https://oss.sonatype.org/content/repositories/snapshots/com/orientechnologies/orientdb-community/3.1.1-SNAPSHOT/

Thanks

Luigi

**Comment by JeffJochems** [comment]
Great! 
Thanks @luigidellaquila and everyone!
